modules/luci-base/htdocs/luci-static/resources/form.js

```diff
@@ -3938,6 +3948,9 @@ var CBIDummyValue = CBIValue.extend(/** @lends LuCI.form.DummyValue.prototype */
 		if (this.href && !((this.readonly != null) ? this.readonly : this.map.readonly))
 			outputEl.appendChild(E('a', { 'href': this.href }));
 
+		if (this.hidden)
+			outputEl = E('div', { 'style': this.hidden ? 'display:none' : null });
```

Comment:
Hmmm. ..
I would guess that jow meant the original outputEl declaration 5 lines above.
(now it looks like you are decalring it twice)

Reply:
Thanks, I think I've got that right now.  So, if we wanted to add more in the future, we can have unlimited parameters there, right? For example:

```
outputEl = E('div', 
    { 'style': this.hidden ? 'display:none' : null },
    { 'style': this.new1 ? 'something' : null },
    { 'style': this.new2 ? 'somethingElse' : null }
);
```